src/azul/queues.py

```diff
@@ -377,19 +380,24 @@ def manage_lambdas(self, queues: Mapping[str, Queue], enable: bool):
             def submit(f, *args, **kwargs):
                 futures.append(tpe.submit(f, *args, **kwargs))
 
+            indexer = load_app_module('indexer')
+            queues_to_lambda_names = {
+                lambda_.queue: [config.indexer_function_name(lambda_.lambda_name)]
```

Comment:
```suggestion
                lambda_.queue: [config.indexer_function_name(lambda_.handler_name)]
```

In a separate commit at the beginning of the history.

Or even better

```suggestion
                lambda_.queue: [config.indexer_function_name(lambda_.name)]
```

Reply:
For the record, I had originally called this attribute `.name` in the first PR, but remember you asking me to change it to `lambda_name` (unfortunately I couldn't find a written record of this).

I'm happy to make this change.